### PR TITLE
Set up database

### PR DESCRIPTION
**Pull request by Creiwry**
- Set up database models 
  - Property
  - UserProperty

- Add first_name and last_name fields to User model
- Set up active storage for Property images
- Create seed for models

### CODE REVIEW

File: app/controllers/properties_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_property
+      @property = Property.find(params[:id])
+    end
+
+    # Only allow a list of trusted parameters through.
+    def property_params
+      params.require(:property).permit(:name, :location, :description, :area, :number_of_rooms, :furnished, :terrace, :basement)
```
Comment (greg-Da) [inline app/controllers/properties_controller.rb:49]: We need a city param at least until we add google map / leaflet

File: db/migrate/20231127115503_create_properties.rb
Comment (greg-Da) [inline db/migrate/20231127115503_create_properties.rb:1]: We need a city param

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,36 @@
...
+    description: Faker::Lorem.paragraphs.join(' '),
+    area: Faker::Number.between(from: 20, to: 200),
+    number_of_rooms:,
+    number_of_bedrooms:,
+    furnished: booleans.sample,
+    terrace: booleans.sample,
+    basement: booleans.sample,
+    renting: booleans.sample,
+    user: users.sample
+  )
```
Comment (greg-Da) [inline db/seeds.rb:39]: We need a city from a specific array

File: app/controllers/properties_controller.rb (hunk 1 above)
Comment (Creiwry) [reply app/controllers/properties_controller.rb:49]: ok, will add

**Review by greg-Da — APPROVED** [review]

**Pull request merged**